ksuid.go

```diff
@@ -311,3 +312,33 @@ func quickSort(a []KSUID, lo int, hi int) {
 		quickSort(a, i+1, hi)
 	}
 }
+
+// Next returns the next KSUID after id.
+func (id KSUID) Next() KSUID {
+	zero := makeUint128(0, 0)
+
+	t := id.Timestamp()
+	u := uint128Payload(id)
+	v := add128(u, makeUint128(0, 1))
```

Comment:
How often will Next()/Prev() be called? is it worth storing the result of `makeUint128(0, 1)` in a variable that gets re-used in Next/Prev?

Reply:
`makeUint128` gets inlined, so I don't think it has any cost, it compiles down to a single instruction. Loading it from a global variable could actually be slower since the compiler won't know this is a constant value anymore.